html/templates/data.html

```diff
@@ -1,41 +1,667 @@
-{% extends "base.html" %}
-{% set active = "data" %}
-
-{% block content %}
+{% extends "base.html" %} {% set active = "data" %} {% block content %}
+<style>
+  .accordion-button {
+    background-color: #f8f9fa;
+    font-size: 1.2rem;
+    font-weight: 600;
+    border-radius: 0.5rem;
+    margin-bottom: 0.5rem;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+    padding: 1rem 1.25rem;
+    padding-right: 5rem;
+    position: relative;
+    display: flex;
+    align-items: center;
+  }
+  .accordion-button::after {
+    order: -1;
+    margin-right: 1rem;
+    margin-left: 0;
+  }
+  .accordion-button .format-label {
+    font-size: 0.9rem;
+    font-weight: normal;
+    position: absolute;
+    right: 1.25rem;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  .accordion-item {
+    border: none;
+  }
+  .accordion-body {
+    padding: 1rem 1.25rem;
+    border-radius: 0.5rem;
+    background-color: #ffffff;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.05);
+  }
+</style>
 <div class="position-relative text-white d-none d-md-block">
-    <img class="img-fluid w-100" src="static/img/quartz.png">
-    <div class="position-absolute top-50 start-5 translate-middle-y">
-        <h1 class="px-4 py-2 fs-large border-start border-2 border-white paint-large-header">Database Overview</h1>
-        <p class="my-3 dynamic-p paint-small-header">Calibration, deflectometry, kinematic, weather</p>
-    </div>
+  <img class="img-fluid w-100" src="static/img/quartz.png" />
+  <div class="position-absolute top-50 start-5 translate-middle-y">
+    <h1
+      class="px-4 py-2 fs-large border-start border-2 border-white paint-large-header"
+    >
+      Database Overview
+    </h1>
+    <p class="my-3 dynamic-p paint-small-header">
+      Calibration, deflectometry, kinematic, weather
+    </p>
+  </div>
 </div>
-
 <h4 class="my-4 py-2 text-primary">Data Repository</h4>
+Our web page offers an <a href="/WRI1030197">HTML-browsable version</a> of the
```

Comment:
These line breaks here are somewhat odd. Are they intentional or are these weirdly rendered on GitHub?

Reply:
If these were intended to get a certain width I would rather work with the overflow CSS attribute